### PR TITLE
updating tests to use new mc --enc flags

### PR DESCRIPTION
**Pull request by zveinn**
## Community Contribution License
All community contributions in this pull request are licensed to the project maintainers
under the terms of the [Apache 2 license](https://www.apache.org/licenses/LICENSE-2.0). 
By creating this pull request I represent that I have the right to license the 
contributions to the project maintainers under the Apache 2 license.

# NOTE
We don't want to merge this until we release a new version of the mc.

## Description
Encryption flags in some of our tests needed updating due to this PR: https://github.com/minio/mc/pull/4882

### CODE REVIEW

**Review by shtripat — APPROVED** [review]
lgtm

**Comment by harshavardhana** [comment]
Test failure looks like the flag is missing @zveinn

**Comment by zveinn** [comment]
> Test failure looks like the flag is missing @zveinn

The test suite is pulling mc from the latest release.. so the flags won't be available until we release the mc changes officially. 

I'm keeping an eye on it. Until then we can just let this sit.

**Review by harshavardhana — APPROVED** [review]

File: docs/bucket/replication/setup_3site_replication.sh [hunk 1]
```diff
@@ -194,10 +194,10 @@ fi
 sleep 5
 
 head -c 221227088 </dev/urandom >200M
-./mc.RELEASE.2021-03-12T03-36-59Z cp --config-dir ~/.mc --encrypt "sitea" --quiet 200M "sitea/bucket/200M-enc-v1"
+./mc.RELEASE.2021-03-12T03-36-59Z cp --config-dir ~/.mc --enc-s3 "sitea" --quiet 200M "sitea/bucket/200M-enc-v1"
```
Comment (harshavardhana) [inline docs/bucket/replication/setup_3site_replication.sh]:
```suggestion
./mc.RELEASE.2021-03-12T03-36-59Z cp --config-dir ~/.mc --encrypt "sitea" --quiet 200M "sitea/bucket/200M-enc-v1"
```

Comment (harshavardhana) [reply docs/bucket/replication/setup_3site_replication.sh]: what I had meant was this ^^ @zveinn

**Pull request merged**